### DISCUSSION TITLE
OpenAndroMaps winter map theme

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#647] [Show and tell]
OpenAndroMaps winter map theme:

https://www.senotto.de/Tipps_Tricks/GPS/OAM_Winter/OAM_Winter.html

**Comment by kds112** [comment]
Will you offer the theme in the download manager?

**Reply by devemux86** [reply]
It is easier if they include it in the OpenAndroMaps server, like the [Voluntary](https://voluntary.nichesite.org/) theme.

Can you suggest it in the [OpenAndroMaps forum](https://www.openandromaps.org/en/oam-forums)?

**Reply by kds112** [reply]
Okay, I did it...
